registration/views.py

```diff
@@ -825,12 +835,19 @@ def form_valid(self, form):
                 form.add_error(field=None, error=error_non_field_names)
             return self.form_invalid(form)
 
-        return super(EmailNextOfKinView, self).form_valid(form)
+        return super(EmailNextOfKinBaseView, self).form_valid(form)
 
     def submit_nok_email(self, data):
         try:
             data_copy = data.copy()
+            data_copy['variant_id'] = self.variant_id
             data_copy['from_email'] = self.request.session.get(SESSION_EMAIL, settings.DEFAULT_FROM_EMAIL)
+            body = "{}{}".format(data_copy['body'], _(
```

Comment:
Not sure if I'm following.
I understand the point that current code has a security leak and I agree we should address it. To make sure I'm understanding your solution, the main idea is to remove any html code in register and let 53 handle any security issue, this means that we should get the body as text and send it 53, 53 will construct html from the text body. But for the link, as you can see, its `register` link, as 53 is a service provider, we can't put link to `register` in 53. So the only case I can think of it is to send two parameters, the `body` and the `link` and 53 will construct the full email.
@danaspiegel what do you think?